src/pyOpenMS/tests/unittests/test_arrow_zerocopy.py

```
@@ -238,3 +248,141 @@ def test_to_arrow_both(self, ms2_experiment):
+
+# ---------------------------------------------------------------------------
+# ArrowSchemaValidation tests
+# ---------------------------------------------------------------------------
+
+class TestValidateArrowSchema:
+
+    def test_strict_valid(self):
+        """A table matching the schema exactly passes strict validation."""
+        schema = FeatureSchema.schema()
+        table = pa.table({name: pa.array([0], type=field.type)
+                          for name, field in zip(schema.names, schema)})
```

Ruff (0.15.6)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)